src/libcore/slice/mod.rs

```diff
@@ -2591,7 +2591,7 @@ impl<'a, T> ExactSizeIterator for ExactChunksMut<'a, T> {
     }
 }
 
-#[unstable(feature = "fused", issue = "35602")]
+#[stable(feature = "fused", since = "1.25.0")]
```

Comment:
`ExactChunksMut` is still unstable so this should be `#[unstable(feature = "exact_chunks", issue = "47115")]`.